app/models/user/time/TimeSpan.scala

```diff
@@ -137,13 +137,14 @@ class TimeSpanDAO @Inject()(sqlClient: SQLClient)(implicit ec: ExecutionContext)
       parsed <- Fox.combined(r.toList.map(parse))
     } yield parsed
 
-  def findAll(start: Option[Long], end: Option[Long]): Fox[List[TimeSpan]] =
+  def findAll(start: Option[Long], end: Option[Long], organizationId: ObjectId): Fox[List[TimeSpan]] =
     for {
-      r <- run(Timespans
-        .filter(r =>
-          notdel(r) && (r.created >= new java.sql.Timestamp(start.getOrElse(0))) && r.created <= new java.sql.Timestamp(
-            end.getOrElse(MAX_TIMESTAMP)))
-        .result)
+      r <- run(sql"""select #${columnsWithPrefix("t.")} from #${existingCollectionName} t
```

Comment:
I' wondering why we can't use the scala slick syntax here. Or do you simply prefer SQL?

Reply:
In my first draft, I was going to use the accessQueries to find all items in the visible scope of the requesting user, rather than filter by organization (that would have required sql syntax). However, it turned out that leads to inconsistencies e.g. with public datasets. So I changed course, but I didn’t change this code back to slick syntax.

However, I think the more complex the query, the easier to write and read is SQL compared to slick syntax. Since we introduced a join here, I’d keep this change.

Reply:
Fine with me :+1: